localization/english/event_localization/wc_central_kalimdor_events_l_english.yml

```diff
@@ -0,0 +1,250 @@
+cen_kal.1005.desc:0 "Quite some time ago, you agreed to invest in a goblin oil project in [ROOT.Var('depot_location').Province.GetBaronyNameExplicitly]. You were promised a considerable share of the income from the oil sales, as well as access to some of the goblin technology and expertise.\n\nFor a while, the project seemed to be going well. The goblins managed to extract and refine large quantities of oil from the ground, and sold it to various buyers and markets. You received a steady stream of gold and goods from them, as well as some benefits for your infrastructure and military. You also gained some prestige and influence among your peers, who admired your vision and courage.\n\nNow, you have received a letter from the goblins second in command - [SCOPE.sC('goblin_prospector').GetFirstName], who claims to have staged a coup and taken control of the oil depot. They say that they have overthrown the former leader - [SCOPE.sC('old_prospector').GetFirstName], who was corrupt and incompetent, and have taken charge of the project. They also claim to be far more trustworthy and efficient than their predecessor, and that they would like to restart the original deal with you. They say that they will retain the agreed share of the income from the oil sales from the old deal but are in need of further investment to repair the damages caused to the depot during the coup.\n\nYou are not sure what to make of this letter. You wonder if this new goblin leader is better or worse than the old one. You wonder how your subjects, vassals, and allies will react to this proposal."
+cen_kal.1002.a:0 "I'm honestly surprised this worked..."
+cen_kal.1002.a.tt:0 "Every year, an event will occur displaying the returns of the oil depot. Oil prospection workload decision unlocked."
+cen_kal.1003.a:0 "Damn goblins..."
+cen_kal.1001.c2:0 "I won't make the same mistake twice!"
+
+#Working Goblin Depot
+cen_kal.1005.a:0 "I accept your terms, goblin..."
+cen_kal.1005.b:0 "I have tolerated your presence in my lands for long enough..."
+cen_kal.1005.b.tt:0 "Have the goblin depot camp assaulted by your levies, while this will end the goblin presence, it also means that the depot will sustain irreversible damage during the attack."
```

Comment:
```suggestion
cen_kal.1005.b.tt:0 "Have the goblin depot camp assaulted by your levies. While this will end the goblin presence, it also means that the depot will sustain irreversible damage during the attack."
```